Cargo.toml

```diff
@@ -71,15 +71,15 @@ tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["env-filter", "json"] }
 futures = "0.3"
 thiserror = "2"
-k8s-openapi = { version = "0.23", features = ["earliest"] }
+k8s-openapi = { version = "0.24", features = ["earliest"] }
 reqwest = { version = "0.12", default-features = false, features = [
     "blocking",
     "rustls-tls-native-roots",
     "json",
     "socks",
     "http2",
 ] }
-kube = { git = "https://github.com/kube-rs/kube.git", rev = "ecbdafc214538aadc78ec8447f2fa12d0057492b", default-features = false, features = [
+kube = { git = "https://github.com/kube-rs/kube.git", tag = "0.98.0", default-features = false, features = [
```

Comment:
can use upstream, no? without git

Reply:
yes, this was mainly a test to see if it removes the advisory